sdk/tables/test-resources.json

```diff
@@ -4,6 +4,13 @@
     "parameters": {
         "baseName": {
             "type": "String"
+        },
+        "cosmosdblocation": {
```

Comment:
Could you clarify why we need a separate location for cosmos than the one we use for storage?